src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__cell calendar__cell--start-day"></div>
```

Comment:
what do you want to do with class `calendar__cell--start-day`?
you are not adding any css for it
also your task is add class like this to .calendar directly
and then depends from it you should change start day for your calendar

Reply:
if you have this class `calendar--start-day_tue`, for example
You should start month from tuesday